### PR TITLE
Remove obsolete BigQuery connector for Hadoop MR

### PR DESCRIPTION
**Pull request by medb**
BigQuery connector for Hadoop MR didn't see a new release for 2 years already and de-facto an obsolete. There new Hive and Spark BigQuery connectors that are actively developed and maintained - they should be used instead:
- https://github.com/GoogleCloudDataproc/hive-bigquery-connector
- https://github.com/GoogleCloudDataproc/spark-bigquery-connector

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,36 +36,22 @@ commands from the main directory:
 ./mvnw -P coverage clean verify
 ```
 
-The Cloud Storage connector JAR can be found in `gcs/target/`. The BigQuery
-connector JAR can be found in `bigquery/target/`.
+The Cloud Storage connector JAR can be found in `gcs/target/` directory.
 
-## Adding the Cloud Storage and BigQuery connectors to your build
+## Adding the Cloud Storage to your build
```
Comment (cjac) [inline README.md:41]: s/Adding the Cloud Storage to/Adding the Cloud Storage connector to/

**Review by cjac — COMMENTED** [review]
I haven't run the tests, but I found a grammar update

File: README.md (hunk 1 above)
Comment (medb) [reply README.md]: Good catch. Done!

**Review by cnauroth — APPROVED** [review]
LGTM

Thank you for the patch, @medb . Thank you @cjac for code reviewing.

**Comment by cjac** [comment]
Grammar looks good.  I didn't see any description of what's going on here.  It looks like the unused and obsolete interface to BigQuery from Map/Reduce is being removed from the code base.  Is there a summary somewhere I can review?

**Comment by medb** [comment]
> Grammar looks good. I didn't see any description of what's going on here. It looks like the unused and obsolete interface to BigQuery from Map/Reduce is being removed from the code base. Is there a summary somewhere I can review?

Added to the PR description.

**Review by cjac — COMMENTED** [review]
LGTM.  If I get support tickets for this API not being available on recent releases, I will advise they pin to an older release and help customers migrate to something modern.

**Review by arunkumarchacko — COMMENTED** [review]
LGTM. Thank you for taking care of this.

**Pull request merged**